dynaboard.yml

```diff
@@ -1366,17 +1593,21 @@ nodes:
           padding-x: 8px;
         }
       value: Fork →
+      showExternalIndicator: '{{false}}'
       fontSize: sm
       textAlign: right
       verticalAlign: bottom
       fontWeight: bold
-      color: "{{isHovered.value ? 'text.primary' : 'text.muted'}}"
+      color: '{{isHovered.value ? ''text.primary'' : ''text.muted''}}'
```

Comment:
@swimmadude66 should this have double single quotes?

Reply:
its just how the yaml got exported. its still valid, but I'll switch it to double quotes for readability